test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -255,13 +304,10 @@ namespace iroha {
                        .getAccount(account->accountId())
                        .build();
       auto result = executeQuery(query);
-      ASSERT_NO_THROW({
-        const auto &cast_resp = boost::apply_visitor(
-            framework::SpecifiedVisitor<
-                shared_model::interface::AccountResponse>(),
-            result->get());
-        ASSERT_EQ(cast_resp.account().accountId(), account->accountId());
-      }) << result->toString();
+      checkSuccessfulResult<shared_model::interface::AccountResponse>(
+          std::move(result), [this](auto &&cast_resp) {
```

Comment:
I believe `const auto &cast_resp` should work well here. I don't see a reason to use `&&` here.